package.json

```diff
@@ -29,29 +29,29 @@
   },
   "homepage": "https://github.com/digicatapult/openapi-service-template#readme",
   "dependencies": {
-    "body-parser": "^1.19.0",
+    "body-parser": "^1.20.0",
     "compression": "^1.7.4",
     "cors": "^2.8.5",
-    "dotenv": "^8.2.0",
-    "envalid": "^7.0.0",
-    "express": "^4.17.1",
-    "express-openapi": "^10.0.0",
-    "pino": "^6.11.1",
-    "pino-http": "^5.5.0",
-    "swagger-ui-express": "^4.2.0"
+    "dotenv": "^16.0.0",
+    "envalid": "^7.3.1",
+    "express": "^4.17.3",
+    "express-openapi": "^10.1.0",
+    "pino": "^7.10.0",
+    "pino-http": "^6.6.0",
+    "swagger-ui-express": "^4.3.0"
   },
   "devDependencies": {
-    "chai": "^4.3.1",
+    "chai": "^4.3.6",
     "chai-json": "^1.0.0",
-    "depcheck": "^1.4.0",
-    "eslint": "^7.21.0",
-    "eslint-config-prettier": "^8.1.0",
-    "eslint-plugin-prettier": "^3.3.1",
-    "mocha": "^9.1.2",
-    "nodemon": "^2.0.7",
+    "depcheck": "^1.4.3",
+    "eslint": "^8.13.0",
```

Comment:
Please verify we comply with the new changes
https://eslint.org/docs/8.0.0/user-guide/migrating-to-8.0.0

Reply:
Looks good to me!